buildlib/azure-pipelines-perf.yml

```diff
@@ -99,6 +99,7 @@ stages:
               module load /hpc/local/etc/modulefiles/hpcx-gcc
 
               set -xeE -o pipefail
+              export SOURCE_BRANCH=$(Build.SourceBranch)
```

Comment:
why needed? who uses this var?

Reply:
if it's perfx.py - maube should pass it by cmdline param?

Reply:
Used by the PerfX code:
https://github.com/Mellanox-lab/PerfX/pull/53

Reply:
Passing as a param is an option, but would require bigger changes at PerfX.
Need to add args parsing, then path it all the way to the library that actually uses it.
I would prefer the current solution.

Reply:
but i see other things are passed as parameters: "--config", "--before", "--after"
why SOURCE_BRANCH is different?

Reply:
Good point, will do. 
I was expecting Azure built-in vars to be available in ENV, like it works with "AGENT_ID".
That's how I started with using it first and exporting later.

Reply:
IMO, PerfX should not expect any Azure env vars, so we could run it also from command line

Reply:
Done